### DISCUSSION TITLE
Unintentional Moving of Commit Window

### DISCUSSION THREAD

**Discussion by jackwellslumiradx** [gitextensions/gitextensions#8608] [Q&A] [answered]
I have some how moved my "commit" view window from the bottom section to next to my history tree. It's very annoying and I can't find any documentation on _how_ to move the windows around. 

How can I get the "Commit" view back down where it belongs?!

![image](https://user-images.githubusercontent.com/59561298/98658813-5f122780-233b-11eb-9a9e-cb5002a7a87f.png)

**Comment by pmiossec** [comment, marked as answer]
Use the fourth button in the toolbar...

**Reply by pmiossec** [reply]
With an image...
![image](https://user-images.githubusercontent.com/460196/98660200-50307280-2345-11eb-911b-d31ea6f30ebc.png)

**Reply by jackwellslumiradx** [reply]
Hallelujah! Thank you! A wee interface guide with the tips in the docs would help! (Unless I missed it?)

**Reply by pmiossec** [reply]
We are looking for help on the doc. So if you think you could improve it, please, feel free to contribute.
That's not so difficult.
You will find the doc repository here: https://github.com/gitextensions/GitExtensionsDoc

**Reply by pmiossec** [reply]
Probably this file to update: https://github.com/gitextensions/GitExtensionsDoc/blob/master/source/browse_repository.rst
